conda/recipes/libcuml/meta.yaml

```diff
@@ -47,6 +48,7 @@ requirements:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
     - treelite=0.92
     - faiss-proc=*=cuda
+    - gtest=1.10.0
```

Comment:
i don’t think we need this as a run dependency

Reply:
Do we need it as a build dependency for the conda package anyway? Tests typically aren't built as part of the package IIRC?

Reply:
I think @kkraus14 you meant "run" dependency above. Agree with you both. This should only be in build dep.

Reply:
I'm pretty sure for conda builds in general we don't build the tests (but I could be mistaken) where we wouldn't even need it as a build dependency either.

Reply:
In the new build process we will build the gtests with conda builds, while we don't pkg them we will use them in the GPU test phase with the artifacts shipped over.

Reply:
The new build process will build the tests as part of the conda package, so definitely need this dependency for building. Probably not for run though.